Leanplum-SDK/Classes/Managers/Networking/LPNetworkEngine.m

```diff
@@ -112,6 +115,10 @@ - (void)setHostName:(NSString *)hostName
     [request setValue:[NSString stringWithFormat:@"%@, en-us",
                        [[NSLocale preferredLanguages] componentsJoinedByString:@", "]]
                         forHTTPHeaderField:@"Accept-Language"];
+    if ([self.featureFlagManager isFeatureFlagEnabled:LP_FEATURE_FLAG_REQUEST_GZIP]) {
+        [request setValue:@"gzip" forHTTPHeaderField:@"Accept-Encoding"];
+        [self.countAggregator incrementCount:@"operation_with_request_gzip"];
+    }
```

Comment:
this is great. 1 addition would make it even better..
let's have  
```suggestion
     } else {
         [self.countAggregator incrementCount:@"operation_without_request_gzip"];
     }
```
so that we can really see the rollout change if we want to